Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/OverFlow/draftMessageOverflow.tsx

```diff
@@ -164,6 +167,14 @@ class Overflow extends React.Component<OverflowProps, OverflowState> {
             this.props.getDraftMessagesList().then(() => {
                 this.props.getMessagesList();
             });
+            this.interval = setInterval(() => {
```

Comment:
I have 2 questions about adding the interval in here:

1). Will it create multiple intervals if user sends multiple notifications within 1 minute?
If so, the intervals may cause many refreshments than expected.

2). The Overflow component is used by DraftMessages component. 
After user clicks the send button, the draft notification will be removed from DraftMessage component immediately.
Will the interval still alive in this case?

Reply:
As the current workflow, it will take time for user to create message, press send button and open confirmation page to send message. so that won't have too many refreshment in 1 minute